lib/msal-angular/docs/v2-docs/errors.md

```diff
@@ -25,7 +25,7 @@ In `@azure/msal-angular` there are 2 common scenarios when this can happen:
 
 We recommend that you bootstrap the the **`MsalRedirectComponent`** if you want to use redirects as this will handle all redirects for you. See the docs on redirects [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/lib/msal-angular/docs/v2-docs/redirects.md) for how to use the `MsalRedirectComponent` or alternatively calling `handleRedirectObservable` manually. 
```

Comment:
This section should also be updated if you make any changes suggested above.

Reply:
Done